client/src/containers/Courses/Courses.js

```diff
@@ -16,6 +16,16 @@ class Courses extends Component {
     const courses = res.data;
     this.setState({ courses });
   };
+  setCourses = clickEvent => {
```

Comment:
Why is this code part of this task? What is it about? I don't think it's related to update organisation, does it?

Reply:
I do not know. It does not relate to my task.

Reply:
Can you remove this code from your PR Shawki so it's not added again. Make sure it's not used before.

Reply:
I think it was for displaying courses for student, but we haven't implemented this yet, so I decided to delete it for now and just show courses for admin first, and then we can come back to student, because student will see courses in different way I suppose